### DISCUSSION TITLE
Installing Make on MacOS

### DISCUSSION THREAD

**Discussion by DuncanDuMond** [Cyfrin/foundry-full-course-cu#387] [Q&A] [answered]
I installed Make on VS Code but get the following prompt:

*** No targets specified and no makefile found.  Stop.

I'm not sure where to go because there is nothing I have seen where I can install it (for Mac users).

**Comment by garyghayrat** [comment, marked as answer]
Hi, you can install it using brew `brew install make`
If you don't have brew already, check it out https://brew.sh/

**Comment by DuncanDuMond** [comment]
Got it to work

Thanks
